widgetslabextension/scripts/buildExtension.js

```diff
@@ -7,11 +7,6 @@ var buildExtension = require('jupyterlab-extension-builder').buildExtension;
 
 buildExtension({
   name: 'widgetslabextension',
-  entryPath: './lib/plugin.js',
-  extractCSS: true,
-  config: {
-      output: {
-        path: path.join(process.cwd(), 'widgetslabextension', 'static'),
-    }
-  }
+  entry: './lib/plugin',
+  outputDir : 'widgetslabextension/static'
```

Comment:
yes. It uses npm's `path.resolve`. Though it's clearer with a leading `./`, so I'll put it in.